WordPress/WordPressTest/Domains/AllDomainsListItemViewModelTests.swift

```diff
@@ -0,0 +1,108 @@
+import XCTest
+
+@testable import WordPress
+
+final class AllDomainsListItemViewModelTests: XCTestCase {
```

Comment:
The test cases were designed in a way to cover all lines of code in `AllDomainsListItemViewModel`. To verify that, enable `Code Coverage` then run the tests.

![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/4518067a-2ef5-4a58-87d1-b21749b208c4)

Reply:
@Mokagio, `AllDomainsListItemViewModel.swift` was originally intended for the Jetpack target. However, I had to include it in the WordPress target to run unit tests. Is there a way to execute tests specifically on the Jetpack target?

Reply:
@salimbraksa fun fact I _just_ posted in our internal devs channel a link to a PR to explore ways to go in that direction.

The PR is https://github.com/wordpress-mobile/gutenberg-mobile/pull/6348 but, unfortunately, it doesn't make what you ask possible. So I guess we'll have to keep the file in both target for a little while.

Thank you for adding tests for this!